pathfinder/tests/fixtures/network_service.py

```diff
@@ -84,13 +84,13 @@ def channel_descriptions_case_2() -> List:
     #       \-- 5 --/
 
     channel_descriptions = [
-        (0, 100,  20, 0.3, 1,  50,  10, 0.3),  # capacities  90 --  60
-        (1,  40,  10, 0.2, 4, 130, 100, 0.2),  # capacities 130 --  40
-        (0,  90,  10, 0.1, 2,  10,   0, 0.1),  # capacities  80 --  10
-        (2,  50,  20, 0.2, 3,  50,  20, 0.2),  # capacities  50 --  50
-        (3, 100,  40, 0.1, 4,  80,   0, 0.1),  # capacities  60 -- 120
-        (2,  30,  10, 0.1, 5,  40,  15, 0.1),  # capacities  35 --  35
-        (5, 500, 900, 0.1, 4, 750, 950, 0.1),  # capacities 550 -- 700
+        (0, 100,  20, 3000, 1,  50,  10, 3000),  # capacities  90 --  60
+        (1,  40,  10, 2000, 4, 130, 100, 2000),  # capacities 130 --  40
+        (0,  90,  10, 1000, 2,  10,   0, 1000),  # capacities  80 --  10
+        (2,  50,  20, 1500, 3,  50,  20, 1500),  # capacities  50 --  50
+        (3, 100,  40, 1000, 4,  80,   0, 1000),  # capacities  60 -- 120
+        (2,  30,  10, 1000, 5,  40,  15, 1000),  # capacities  35 --  35
+        (5, 500, 900, 1000, 4, 750, 950, 1000),  # capacities 550 -- 700
     ]
```

Comment:
In these two blocks the fee refactoring is inconsistent. We currently referenced the variable as `percentage_fee`. Following this, renaming it to `ppm_fee` and a multiplication by 10^4 would keep it consistent.

Reply:
Sure it's not consistent, but does it matter!? The numbers are made up anyways.

Reply:
No doesn’t matter, but with consistent multiplication tests won't fail !

Reply:
How do you mean "test won't fail"?

Reply:
We have to recompute the results for every assertion, but anyhow there's different ways to do this correctly. 
I'd still suggest to change percentage_fee to ppm_fee.